apf/apfMixedVectorElement.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2011 Scientific Computation Research Center
+ *
+ * This work is open source software, licensed under the terms of the
+ * BSD license as described in the LICENSE file in the top-level directory.
+ */
+
+#ifndef APFMIXEDVECTORELEMENT_H
+#define APFMIXEDVECTORELEMENT_H
+
+#include "apfElementOf.h"
+#include "apfMatrix.h"
+
+namespace apf {
+
+class MixedVectorField;
+
+class MixedVectorElement : public ElementOf<Vector3, double>
```

Comment:
a brief description of what this class does/supports would be helpful

Reply:
Added explanation in the new rebased branch.